packages/client/e2e/pages/FxNewRfqPageObject.ts

```diff
@@ -0,0 +1,31 @@
+import { Page, TestInfo } from "@playwright/test"
+
+export class FxNewRfqPageObject {
+  constructor(
+    readonly page: Page,
+    readonly testInfo: TestInfo,
+  ) {}
+
+  async clickCurrency(currency: string) {
```

Comment:
as credit, 
```suggestion
  async selectFilter(filter: string) {  
```

Reply:
but should not be marked `async` - neither should the originals in the Credit POs - did we discuss this already?